contracts/interfaces/IStakingPool.sol

```diff
@@ -18,6 +18,8 @@ struct AllocationRequest {
   uint capacityReductionRatio;
   uint rewardRatio;
   uint globalMinPrice;
+  uint32 newPeriod;
+  uint32 remainingPeriod;
```

Comment:
do you really need this here? you can technically deduct it based on `previousExpiration`